magento/magento2
# Fixed Issue Configurable product not showing price

Fixed Issue: Configurable product not showing price on category and product page #28068: it was not showing price when all simple products are out of stock.





### Description (*)
Configurable products not showing price on category and product page when all the associated simple products are out of stock.


### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2# https://github.com/magento/magento2/issues/28068

### Manual testing scenarios (*)


    1. Create a configurable product with all its associated simple products as out of stock
    2. On front-end go to product page of that configurable product
    3. The product price should be shown at product page


### Questions or comments


### Contribution checklist (*)
